ucp/core.py

```diff
@@ -581,6 +587,32 @@ async def send(self, buffer, tag=None):
             tag += self._send_count
         return await comm.tag_send(self._ep, buffer, nbytes, tag, name=log)
 
+    @nvtx_annotate("UCXPY_AM_SEND", color="green", domain="ucxpy")
+    async def am_send(self, buffer):
+        """Send `buffer` to connected peer.
+
+        Parameters
+        ----------
+        buffer: exposing the buffer protocol or array/cuda interface
+            The buffer to send. Raise ValueError if buffer is smaller
+            than nbytes.
+        """
+        if self.closed():
+            raise UCXCloseError("Endpoint closed")
+        if not isinstance(buffer, Array):
+            buffer = Array(buffer)
+        nbytes = buffer.nbytes
+        log = "[Send #%03d] ep: %s, tag: %s, nbytes: %d, type: %s" % (
```

Comment:
```suggestion
        log = "[AM Send #%03d] ep: %s, tag: %s, nbytes: %d, type: %s" % (
```

Reply:
Fixed in https://github.com/rapidsai/ucx-py/pull/701/commits/a3f3b0870d650785cb7bb14993a2da6ee2aaf47e